Marlin/Marlin_main.cpp

```diff
@@ -1635,7 +1630,7 @@ static void clean_up_after_endstop_or_probe_move() {
    *  Plan a move to (X, Y, Z) and set the current_position
    *  The final current_position may not be the one that was requested
    */
-  static void do_blocking_move_to(float x, float y, float z) {
+  static void do_blocking_move_to(float x, float y, float z, float feed_rate = 0.0) {
```

Comment:
Maybe you have not seen that the `feed_rate` parameter is optional and defaults to 0.0. So the current behavior is not changed when omitted. 0.0 is perfect because a move with no speed will never arrive at it's target. Requesting 0.0 by will is extremely unlikely.